From f88a92426528f87d17937e2714810130ab9a3f5a Mon Sep 17 00:00:00 2001
From: Stefan Agner <stefan.agner@toradex.com>
Date: Thu, 18 Jan 2018 10:09:14 +0100
Subject: [PATCH 1/2] add board support for DCD memory timings

Signed-off-by: Stefan Agner <stefan.agner@toradex.com>
---
 iMX8QM/soc.mak | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/iMX8QM/soc.mak b/iMX8QM/soc.mak
index ec7938f..d78b3c0 100755
--- a/iMX8QM/soc.mak
+++ b/iMX8QM/soc.mak
@@ -1,9 +1,14 @@
 MKIMG = ../mkimage_imx8
 #DCD_CFG_SRC = imx8qm_dcd_800MHz.cfg
 #DCD_CFG_SRC = imx8qm_dcd_1.2GHz.cfg
-DCD_CFG_SRC = imx8qm_dcd_1.6GHz.cfg
+ifeq ($(DCD_BOARD),)
+	DCD_CFG_SRC = imx8qm_dcd_1.6GHz.cfg
+	DCD_CFG = imx8qm_dcd.cfg.tmp
+else
+	DCD_CFG_SRC = imx8qm_$(DCD_BOARD)_dcd_1.6GHz.cfg
+	DCD_CFG = imx8qm_$(DCD_BOARD)_dcd.cfg.tmp
+endif
 
-DCD_CFG = imx8qm_dcd.cfg.tmp
 
 CC ?= gcc
 INCLUDE = ./lib
-- 
2.15.1

